### PR TITLE
Update v2.2

### LINKED ISSUE #119: Update on v2.2
Add to model_updates page
- Explainer on the regular day/night attenders issue
- bugfix - day procedures step_counts https://github.com/The-Strategy-Unit/nhp_model/issues/187
- Update data log #59

### PR DESCRIPTION
**Pull request by StatsRhian**
Closes #119 & #120

### CODE REVIEW

File: project_plan_and_summary/model_updates.qmd [hunk 1]
```diff
@@ -3,6 +3,34 @@ title: "Model updates"
...
+
+# Update 2024/08/24
+
+## v2.2
+
+### New point of delivery categories for regular attenders
+
+Regular day/night attenders are patients admitted electively, as part of a planned series of regular admissions for an on-going regime of broadly similar treatment and who is discharged the same day. Intermittent dialysis cases and patients on regular chemotherapy/radiotherapy are examples of regular day admissions.
+
+In v2.1 we updated our definition for last episode in spell to match guidance from NHS digital. When transitioning to this new logic some regular day/night admissions were not captured. This means for some trusts these admissions were missing from the results, depending on how the trust captured these attendees in the baseline data.
```
Comment (StatsRhian) [inline project_plan_and_summary/model_updates.qmd:16]: @yiwen-h Would be nice to the announcement when we changed this but I couldn't find it

Comment (StatsRhian) [reply project_plan_and_summary/model_updates.qmd:16]: Oh I've found the description - just not in v2.1 updates

https://connect.strategyunitwm.nhs.uk/nhp/project_information/data_extraction/data_log.html#last-episode-in-spell

I'll link to that description

Comment (yiwen-h) [reply project_plan_and_summary/model_updates.qmd:16]: Here it is: https://connect.strategyunitwm.nhs.uk/nhp/project_information/data_extraction/data_log.html#last-episode-in-spell

File: project_plan_and_summary/model_updates.qmd [hunk 2]
```diff
@@ -3,6 +3,34 @@ title: "Model updates"
...
 
+
+
+# Update 2024/08/24
+
+## v2.2
+
+### New point of delivery categories for regular attenders
+
+Regular day/night attenders are patients admitted electively, as part of a planned series of regular admissions for an on-going regime of broadly similar treatment and who is discharged the same day. Intermittent dialysis cases and patients on regular chemotherapy/radiotherapy are examples of regular day admissions.
```
Comment (yiwen-h) [inline project_plan_and_summary/model_updates.qmd:14]: There's something funny about the grammar here - "patients admitted electively" is plural but then "who is discharged the same day" is singular

Comment (yiwen-h) [inline project_plan_and_summary/model_updates.qmd:14]:
```suggestion
Regular day/night attenders are patients admitted electively, as part of a planned series of regular admissions for an on-going regime of broadly similar treatment and who are discharged the same day. Intermittent dialysis cases and patients on regular chemotherapy/radiotherapy are examples of regular day admissions.
```

File: project_plan_and_summary/model_updates.qmd [hunk 3]
```diff
@@ -3,6 +3,34 @@ title: "Model updates"
...
+
+In v2.1 we updated our definition for last episode in spell to match guidance from NHS digital. When transitioning to this new logic some regular day/night admissions were not captured. This means for some trusts these admissions were missing from the results, depending on how the trust captured these attendees in the baseline data.
+
+To resolve this we have added two new point-of-delivery categories:
+
+- Regular day attender admission
+- Regular nights attender admission
```
Comment (yiwen-h) [inline project_plan_and_summary/model_updates.qmd:21]:
```suggestion
- Regular night attender admission
```

File: project_plan_and_summary/model_updates.qmd [hunk 4]
```diff
@@ -3,6 +3,34 @@ title: "Model updates"
...
+
+These will appear alongside the other POD options in the Outputs app.
+
+Trusts who have been significantly impacted by the update have been contacted by their Model Relationship Managers. 
+
+We have also expanded our QA process to include a broader set of test data to run our data comparison checks on.
+
+### Other updates
+
+We fixed a small bug which gave an incorrect value of beddays in step count for day cases.
```
Comment (yiwen-h) [inline project_plan_and_summary/model_updates.qmd:31]:
```suggestion
We fixed a small bug which gave an incorrect value of beddays in the step counts/impact of changes for elective activity converted to day case by the [day procedures mitigators](https://connect.strategyunitwm.nhs.uk/nhp/project_information/modelling_methodology/activity_mitigators/inpatient_activity_mitigators.html#day-procedures). 
```

**Review by yiwen-h — CHANGES_REQUESTED** [review]

File: project_plan_and_summary/model_updates.qmd (hunk 4 above)
Comment (StatsRhian) [reply project_plan_and_summary/model_updates.qmd:31]:
```suggestion
We fixed a small bug which gave an incorrect value of beddays in the step counts/impact of changes for elective activity converted to day case by the [day procedures mitigators](../modelling_methodology/activity_mitigators/inpatient_activity_mitigators.html#day-procedures). 
```

**Comment by yiwen-h** [comment]
Sorry, one very last point

We never say anywhere how we define the new pods - should we say in the model update or in the data log? Basically regular day attenders are patients with classpat 3 and regular night attenders are patients with classpat 4

**Review by yiwen-h — APPROVED** [review]

**Review by yiwen-h — APPROVED** [review]

**Comment by StatsRhian** [comment]
> Sorry, one very last point
> 
> We never say anywhere how we define the new pods - should we say in the model update or in the data log? Basically regular day attenders are patients with classpat 3 and regular night attenders are patients with classpat 4

Neither? Feels like something we should define in the project site more generally rather than an update.

I don't see the other PODS defined anywhere. I could add to [Glossary and definitions](https://connect.strategyunitwm.nhs.uk/nhp/project_information/user_guide/glossary.html) for now?

**Pull request merged**